lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -3519,30 +3537,14 @@ def test_url_generation(self):
 class MFERedirectTests(BaseViewsTestCase):  # lint-amnesty, pylint: disable=missing-class-docstring
     MODULESTORE = TEST_DATA_SPLIT_MODULESTORE
 
-    def _get_urls(self):  # lint-amnesty, pylint: disable=missing-function-docstring
-        lms_url = reverse(
-            'courseware_section',
-            kwargs={
-                'course_id': str(self.course_key),
-                'chapter': str(self.chapter.location.block_id),
-                'section': str(self.section2.location.block_id),
-            }
-        )
-        mfe_url = '{}/course/{}/{}'.format(
-            settings.LEARNING_MICROFRONTEND_URL,
-            self.course_key,
-            self.section2.location
-        )
-        return lms_url, mfe_url
-
     def test_learner_redirect(self):
         # learners will be redirected when the waffle flag is set
-        lms_url, mfe_url = self._get_urls()
+        lms_url, mfe_url, preview_url = self._get_urls()  # lint-amnesty, pylint: disable=unused-variable
```

Comment:
Nit: Typically, we'd use:
`lms_url, mfe_url, __ = self._get_urls()`
in this situation, as it declutters the code to remove the unused var name and use `__` instead.